packages/styled-base/src/utils.js

```diff
@@ -11,7 +10,7 @@ export type StyledOptions = {
   target?: string
 }
 
-export type StyledComponent<P> = React.StatelessFunctionalComponent<P> & {
+export type StyledComponent<P> = StatelessFunctionalComponent<P> & {
```

Comment:
I know this isn't from this PR but could we rename `P` to `Props`(and the same thing in the other places we have `P`)?

Reply:
`P` is the same name used by the built-in Flow types for `StatelessFunctionalComponent`, are you sure you want to give it a different name in our implementation?

Reply:
Yes

Reply:
Updated